### ISSUE TITLE
Using editable mission screen for MOTD changes

### ISSUE THREAD

**Issue by Spcemarine** [midspace/Space-Engineers-Admin-script-mod#56]
When my PR gets accepted, I'll use the editable mission screen for setting and changing the MOTD as described in #35.

**Comment by midspace** [comment]
I had a very brief look at the code, and it makes sense.

Sometimes I think a Mission screen just need to be able to present mission text which allows the user to copy detail from it. (like GPS coordinates).

And sometimes in our case, we need to allow the user to enter detail.

I still think it needs more buttons options. Ie., Okay, Cancel, Yes, No.

**Comment by Spcemarine** [comment]
I totally agree with you. That there is only one button is bothering me since it was released. Maybe i'm going to change that to at least two buttons but not yet, other things are more important.
